root.sh

```diff
@@ -99,7 +101,7 @@ else
         -Dsoversion=ON                                            \
         -Dshadowpw=OFF                                            \
         -Dvdt=ON                                                  \
-        -DCMAKE_PREFIX_PATH="${FREETYPE_ROOT}"
+        -DCMAKE_PREFIX_PATH="$FREETYPE_ROOT;$SYS_OPENSSL_ROOT;$GSL_ROOT;$ALIEN_RUNTIME_ROOT;$PYTHON_ROOT;$PYTHON_MODULES_ROOT"
```

Comment:
Yes - and this actually fixes the problem without patching ROOT. (ftr there was a problem in my private config that prevented it from working)